problems/2.functions.js

```diff
@@ -2,16 +2,16 @@
 * @param {string} name - The name of the user to greet
 * @returns {null}
 */
-function logGreeting() {
-
+function logGreeting(name) {
+  console.log("Hello " + name + "!")
 }
 
 /** Returns a custom greeting string in the form "Hello Avery!"
 * @param {string} name - The name of the user to greet
 * @returns {string} - The custom greeting
 */
-function greeting() {
-
+function greeting(name) {
+return "Hello " + name + "!"
```

Comment:
These are both great. I'm going to challenge you to try this again with string iterpolation